meta.yaml

```diff
@@ -16,38 +16,38 @@ requirements:
   host:
     - gensim >=3.4.0
     - matplotlib >=2.2
-    - networkx >=2.2,<2.4
+    - networkx >=2.2
     - numpy >=1.14
     - pandas >=0.24
     - pip
     - python
-    - scikit-learn >=0.20
+    - scikit_learn >=0.20
```

Comment:
Shouldn't this still be kebab case?
```suggestion
    - scikit-learn >=0.20
```
and for line 34 too

Reply:
this form actually came out of the automated `conda skeleton pypi stellargraph` but have fixed now.